src/module.c

```diff
@@ -4850,6 +4850,11 @@ void moduleReleaseGIL(void) {
  *  - REDISMODULE_NOTIFY_STREAM: Stream events
  *  - REDISMODULE_NOTIFY_KEYMISS: Key-miss events
  *  - REDISMODULE_NOTIFY_ALL: All events (Excluding REDISMODULE_NOTIFY_KEYMISS)
+ *  - REDISMODULE_NOTIFY_LOADED: special notification available only for modules,
+ *                               indicate that the key was loaded from rdb.
```

Comment:
```suggestion
 *                               indicates that the key was loaded from persistence.
```

Is it only RDB? I mean, for sure AOF with a preamble will trigger it too, so let's be vaguer.